personal-growth/becoming-a-mentor.md

```diff
@@ -0,0 +1,118 @@
+even though you are a good fit. If that's the case, just be patient and wait for the next hires!
+
+As a mentor, you will be assigned up to 4 mentees and 3 projects that you will manage. Of course,
+this will happen gradually, so that you have the time to get comfortable with your new role and
+responsibilities. Usually, individual contributors who become mentors are assigned to the client
+and team they are already working with, since it allows them to practice their mentoring skills in
+an environment they already know.
+
+You will still hold regular 1:1s with a member of the leadership team, who will follow your progress
+and give you advice on how to up your mentoring game. Mentors need mentors too, after all!
+
+## Running one-on-ones
+
+The tricky thing about one-on-ones is there is no right way to do them: every mentor has their 
+style, which means two one-on-ones can be different even when they're done in the same company. As a
+mentor, you are encouraged to find your unique style and voice.
+
+With that said, there are some ground rules to follow for holding effective one-on-ones (i.e.
+one-one-ones where people say what they want to say rather than what the manager wants to hear):
+
+- **It's about them, not you.** Even though the mentor schedules them, a one-on-one is about the
+  mentee, not the mentor. Let them define the agenda or not have an agenda at all, and let them
+  talk about whatever they want.
+- **Be interested and show it.** Your main job during a one-on-one is to be genuinely interested in
+  what the other person is saying, and to dig deeper and deeper until you are sure all the topics
+  have been discussed exhaustively. 
+- **Do not reschedule.** You should avoid rescheduling a one-on-one like the plague. Of course,
+  we're humans and things happen, so if you absolutely need to, go ahead and reschedule the meeting
+  as close as possible to the original date. 
+- **Create accountability.** It is extremely important for every one-on-one where problems have been
+  raised to create an action plan. Make sure both you and your mentee are accountable for that plan,
+  and monitor its progress closely.
+
+One more tip on scheduling the calls: when you have many mentees, it can be tempting to schedule all
+of the one-on-ones on Friday, so that the rest of your week is free to do other work. However, as
+these meetings are very demanding, it is hard to really focus on more than one or two in the same
+day. Instead, try to spread them out throughout the week, so that you can give the same level of
+attention to the same one-on-one.
+
+## Giving feedback
+
+- Don't wait until the 1:1, give feedback immediately
+- Praise in public, criticize in private
+- Make it about their growth, not their job/responsibilities
+- Listen to the response: sometimes people have good reasons for behaving the way they do
+  - If someone misses all standups for a project perhaps the standups are useless and should be
```

Comment:
Is this an example? If yes, it's not super clear. What about saying that explicitly?